### ISSUE TITLE
Release version should not update until upload completes

### ISSUE THREAD

**Issue by namespacestd0** [biothings/biothings.api#115]
Need to verify the issue first.

**Comment by juliamullen** [comment]
e.g., running this plugin: https://github.com/juliamullen/genomics_api/

with this build config:

```
{
    "build_version": "%Y%m%d%H%M",
    "num_shards": 1,
    "autobuild": {
        "schedule": "2 * * * *",
        "type": "snapshot",
        "env": "auto3"
    },
    "autopublish": {
        "type": "snapshot",
        "env": "auto3"
    }
}
```

will result in the build being triggered before the plugin is finished uploading.

**Comment by newgene** [comment]
should be fixed in #123. 
@juliamullen please verify and close if it works now.
